source/blog/2017-03-22-the-journey-of-a-service-in-rdo.html.md

```diff
@@ -0,0 +1,105 @@
+---
+title: The journey of a new OpenStack service in RDO
+author: amoralej
+tags:
+date: 2017-03-22 19:22:55 CET
+---
+
+
+
+When new contributors join RDO, they ask for recommendations about
+how to add new services and help RDO users to adopt it. This post is
+not a official policy document nor a detailed description about how to carry
+out some activities but provides some high level recommendations to newcomers
+based on what I have learned and observed in the last year working in RDO.
+
+Note that you are not required to follow all these steps and even you can
+have your own ideas about it. If you want to discuss it, let us know your thoughts, we are always open to improvements.
+
+### 1. Adding the package to RDO
+
+The first step is to add the package(s) to RDO repositories as shown
+in [RDO documentation](https://www.rdoproject.org/documentation/rdo-packaging/#how-to-add-a-new-package-to-rdo-trunk).
+This tipically includes the main service package, client library and maybe
+a package with a plugin for horizon. In some cases you may need to add new
+libraries to Fedora which are required for the new packages.
```

Comment:
I'd add some detail here, explaining why we add non-OpenStack, common dependencies to Fedora first.